node/core/approval-voting/src/criteria.rs

```diff
@@ -631,10 +632,10 @@ mod tests {
 					Sr25519Keyring::Bob,
 					Sr25519Keyring::Charlie,
 				]),
-				validator_groups: vec![
+				validator_groups: TypeVec::from(vec![
```

Comment:
we should consider adding a custom macro that resembles the ergonomy of `vec![..]`